test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -13832,10 +13832,10 @@ def add_mom_dyn_res(self, custom_resource, script_body=None,
         else:
             if dirname is None:
                 dirname = self.pbs_conf['PBS_HOME']
+            local_host = socket.gethostname()
             tmp_file = self.du.create_temp_file(prefix=prefix, suffix=suffix,
                                                 body=script_body,
-                                                hostname=host)
-
+                                                hostname=local_host)
             res_file = os.path.join(dirname, tmp_file.split(os.path.sep)[-1])
             self.du.run_copy(host, tmp_file, res_file, sudo=True,
```

Comment:
cp command will fail if given `host` is localhost:
```
$ cp /tmp/1 /tmp/1
cp: '/tmp/1' and '/tmp/1' are the same file
$ echo $?
1
```

Reply:
It doesn't fail because create_teno_file will create file in /tmp/ and in cp command we are copying to either PBS_HOME directory user given directory.